GAME_DESIGN_PRINCIPLES.md

```diff
@@ -0,0 +1,97 @@
+
+### No Generic Creatures
+(WIP)
+
+### No Passive Animals
+Instead of animals standing by to let you kill them, they will run from the player, attack back, and use defense mechanisms. This should make hunting and taming much more engaging.
+
+### No Unbreakable Tools
+In Minecraft, a tool is an investment you can make. You can enchant the tool, rename it, make it your own; there's an incentive to make it last forever with the use of mending. In Cubyz though, you are not creating an investment, but rather a tool to use until it breaks. If a player gets too attached to their tool, then they won't want to make other types of tools, and lower-tier materials will see less use as they will never be used in tool-making.
+
+# Player Engagement
+### "How would the player feel?"
+This is a very important part of game design in general. When a player encounters and re-encounters a mechanic or feature, put yourself in the player's shoes and ask "how would the player feel?" The goal is to make the player feel what you want them to feel, whether that be satisfaction, frustration, excitement, fear, or all of the above! It also helps to ask for feedback from players and asking how they feel about your addition.
+
+### Make the World Feel Alive
+
+### Depth in Simplicity
+Cubyz' special sauce is its simplicity. Keeping everything simple on the surface makes the game approachable for beginners, while the hidden depth keeps it interesting for skilled players. Avoid features that would add unnecessary tedium to the player's experience. Always think about how it would effect the player's first impressions of the game.
+
+### Fuel the Player's Curiosity
+The world is filled with secrets, and we want the player to find these secrets on their own without any outside help or guide.
+
+### Problems have Multiple Solutions
+Problems faced in a particular age of progression should have multiple solutions, instead of just one solution for everything.
+For example, in the "Pre-Caves Age," the player will not have access to coal, as coal spawns low in the world and is shrouded in darkness. There's many solutions to this, however:
+- Find a dim, above ground light source.
+- Search for a cave with exposed coal.
+- Wander into the darkness to find a cave that's bright enough to mine in.
+- Find an above-ground structure that can grant the player coal.
+- Dig straight down until you find coal.
+
+Of course, this problem is completely solved as soon as the player gets coal, as they can now explore caves to find more coal with their newfound torches, but it's important that the player has these options in the first place so they can have more freedom. Not one option is better or easier than the other, and if one turns out to be, then it should be allowed to be adjusted to encourage other methods.
```

Comment:
```suggestion
Of course, this problem is completely solved as soon as the player gets coal, as they can now explore caves to find more coal with their newfound torches, but it's important that the player has these options in the first place to prevent them from getting stuck on a seemingly insurmountable barrier. This also gives them more freedom in how they can approach the game. Not one option is better or easier than the other, and if one turns out to be, then it should be allowed to be adjusted to encourage other methods.
```